### ISSUE TITLE
Maintenence TODO list

### ISSUE THREAD

**Issue by YutackPark** [MDIL-SNU/SevenNet#138]
Updated (3/8) to rearragne develop issues myself. (Not about features)

- [x] numpy >= 2.0.0
- [x] torch >= 2.5.0 (break when TorchScript is related
- [x] torch >= 2.6.0
- [x] Document useful, latest features to README.md or presets. (passing kwargs, multiple testsets, ...)
- [ ] #107 
- [ ] #170 
- [x] Use atoms.rattle() for EFS unit test for stable numerical comparision (especially force)
- [ ] double precision implement
- [ ] training gradient unit test after double precision implement
- [ ] Check parallel features && make compatible with original SevenNet parallel impl
- [ ] Omni, multimodal model training script @Jaesun0912

LAMMPS
- [ ] Test + develop to remove LAMMPS version pin (ML-IAP compatible version)
- [ ] LAMMPS unit test in git actions

**Comment by janosh** [comment]
came here to open an issue for the first 2, then found this issue. thanks for tracking. would be great to be able to use `sevenn` with `numpy>2` and `torch>2.5` 👍
